x-pack/plugins/canvas/canvas_plugin_src/uis/models/metric_vis.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { get } from 'lodash';
+
+import { ViewStrings } from '../../../i18n';
+import { getState, getValue } from '../../../public/lib/resolved_arg';
+
+const { MetricVis: strings } = ViewStrings;
+
+export const metricVis = () => ({
+  name: 'metricVis',
+  displayName: strings.getDisplayName(),
+  args: [
+    {
+      name: 'metric',
+      displayName: strings.getMetricColumnDisplayName(),
+      help: strings.getMetricColumnHelp(),
+      argType: 'vis_dimension',
+      multi: true,
+      default: `{visdimension}`,
+    },
+    {
+      name: 'bucket',
+      displayName: strings.getBucketColumnDisplayName(),
+      help: strings.getBucketColumnHelp(),
+      argType: 'vis_dimension',
+      default: `{visdimension}`,
+    },
+    {
+      name: 'palette',
+      argType: 'extended_palette',
+      options: {
+        type: 'stops',
+      },
+    },
+    {
+      name: 'font',
+      displayName: strings.getFontColumnDisplayName(),
+      help: strings.getFontColumnHelp(),
+      argType: 'font',
+      default: `{font size=60 align="center"}`,
+    },
+    {
+      name: 'colorMode',
+      displayName: strings.getColorModeColumnDisplayName(),
+      help: strings.getColorModeColumnHelp(),
+      argType: 'select',
+      default: 'single',
```

Comment:
Can we default this to be "Labels"? I think that's what the user would expect?

Reply:
Done.